lib/db.js

```diff
@@ -501,17 +500,16 @@ module.exports = (
     )
   }
 
-  DB.prototype.updateSessionToken = function (token, userAgentString, ip) {
+  DB.prototype.updateSessionToken = function (token, ip) {
     log.trace({ op: 'DB.updateSessionToken', uid: token && token.uid })
 
     const uid = token.uid
     if (! this.redis || ! features.isLastAccessTimeEnabledForUser(uid)) {
       return P.resolve()
     }
 
-    token.update(userAgentString)
     const newToken = {
-      tokenId: token.id,
+      tokenId: token.tokenId,
```

Comment:
Changed these references to `id` for the benefit of the local tests, which mostly define mock tokens that only have the `tokenId` property.